userspace/libsinsp/parsers.cpp

```diff
@@ -1448,6 +1457,34 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 	return;
 }
 
+void sinsp_parser::parse_execve_enter(sinsp_evt *evt)
+{
+	sinsp_evt_param *parinfo;
+	char fullpath[SCAP_MAX_PATH_SIZE];
+	uint16_t etype = evt->get_type();
+
+	switch(etype)
+	{
+		case PPME_SYSCALL_EXECVE_8_E:
+		case PPME_SYSCALL_EXECVE_13_E:
+		case PPME_SYSCALL_EXECVE_14_E:
```

Comment:
It seems like `parse_execve_enter` is not invoked with those older events at all, so they could be generalized in the `default` case below, or alternatively you could include them in the `process_event` switch. Not a big deal, but a static analyzer could complain that this part of the code is never reached.